tests/unit/src/test/resources/foldingRange/expect/PartialFunctionCases.scala

```diff
@@ -8,6 +8,7 @@ class A >>region>>{
         println()
         println()
         println()
+        println()
```

Comment:
The threshold is 3, so we should have 4 statements for the fold region to show up.